CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ the APIs in these libraries are stable, and are ready for production use.
 
 ### Updated Libraries
 
+- [Storage Control API](/google/cloud/storagecontrol/README.md) is now GA.
```

Comment:
nit: I  think experimental -> GA belongs in "New Libraries"

Reply:
Done.